avro_schema/init.lua

```diff
@@ -460,79 +468,46 @@ local function compile(...)
             xflatten          = process_lua.xflatten,
             flatten_msgpack   = process_msgpack.flatten,
             unflatten_msgpack = process_msgpack.unflatten,
-            xflatten_msgpack  = process_msgpack.xflatten
+            xflatten_msgpack  = process_msgpack.xflatten,
+            get_names         = function ()
+                return get_names(schema_to_handler, service_fields)
```

Comment:
I would use `list[#list]`, because the name `schema_to_handler` looks confusing for me:

1. It is just handle, but not some map.
2. It does not show explicitly that it is the last schema of provided (in case of two schemas).

I don’t insist.

Reply:
1. `list[#list]` would not work, because it is already converted to a table, and I have to pass a handle, because I have saved backward-compatible methods `get_names` `get_types`, and reuse code. So a handle should be passed.
2. I may pass `args[#args]`, however, IMHO the named variable `schema_to_handler` is easier to understand. Possibly I have to rename it.

Reply:
I’m okay to rename or leave as you wish.

Reply:
renamed to `handler_schema_to`